FlashApp.sh

```diff
@@ -0,0 +1,247 @@
+#!/bin/sh
+
+# Check and if exists, source these files /etc/include.properties and /etc/device.properties
+if [ -f /etc/include.properties ]; then
+    . /etc/include.properties
+fi
+if [ -f /etc/device.properties ]; then
+    . /etc/device.properties
+fi
+
+if [ -z "$PERSISTENT_PATH" ]; then
+    PERSISTENT_PATH="/opt"
+fi
+
+if [ -z "$LOG_PATH" ]; then
+    LOG_PATH="/opt/logs"
+    mkdir -p $LOG_PATH
+fi
+
+FLASHAPPLOGFILE="$LOG_PATH/FlashApp.log"
+EXTBLOCK="$PERSISTENT_PATH/ota/extblock"
+WICIMAGEFILE="$PERSISTENT_PATH/ota/wicimage"
+
+logger() {
+    echo "$(date) FlashApp.sh > $1" | tee -a $FLASHAPPLOGFILE
+}
+
+# Check if all the commands used in this script are available
+commandsRequired="date echo exit ls grep mkdir tar cp rm sync stat losetup mount umount reboot df sed awk md5sum trap"
```

Comment:
Is losetup utility present by default ?

Reply:
Yes; all these commands are now available.